python/doc/user_manual/statistics_on_sample.rst

```diff
@@ -165,6 +165,13 @@ Refer to :ref:`sensitivity_sobol`.
 
     :template: class.rst_t
 
+    HSICStat
+    HSICUStat
+    HSICVStat
+    HSICEstimator
+    HSICEstimatorConditionalSensitivity
+    HSICEstimatorGlobalSensitivity
+    HSICEstimatorTargetSensitivity
     SobolIndicesExperiment
```

Comment:
adding a separate subcategory/toctree would allow to distinguish the sobol classes from the hsic classes more clearly

Reply:
OK, done